### DISCUSSION TITLE
i just got mine very 1st 2XCH !!!!! in 14 days farming @200 plots.. But now cannot sync.. please help

### DISCUSSION THREAD

**Discussion by andy118z** [Chia-Network/chia-blockchain#4471] [Support]
![image](https://user-images.githubusercontent.com/83979862/117707508-3e029480-b201-11eb-91a6-ddadb8fa4079.png)

cannot be sync now.. even i added the node.chia.net or introducer port 

and i already open 8444 port...

anyone can help?

**Comment by SackofFluor** [comment]
congrats!

Did you try to manuelly connect to chiadb.net :8444 ?
For me that often helps speed up the synchronization.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
